### PR TITLE
Fe design modifications

### PR DESCRIPTION
**Pull request by jhubbard2017**

### CODE REVIEW

File: src/app/components/styles/styles.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+export const textfieldStyle = {
```
Comment (lmartin73) [inline src/app/components/styles/styles.js:1]: is is necessary to define this?

File: src/app/routes/auth/containers/Forgot.js [hunk 2]
```diff
@@ -6,16 +6,20 @@ import DisplayContent from '../components/DisplayContent'
 import Footer from '../components/Footer'
 import * as firebase from 'firebase'
 import { smallAlertMessage } from "../../../components/alert-messaging/AlertMessaging";
+import { textfieldStyle } from '../../../components/styles/styles'
 
 
 const mapStateToProps = (state) => {
-    return {}
+    return {
+        state: state
```
Comment (lmartin73) [inline src/app/routes/auth/containers/Forgot.js]: do you really need to define the whole store to the state?

File: src/app/routes/profile/components/profile.js [hunk 3]
```diff
@@ -1,75 +1,38 @@
...
-                                </div>
-                            </div>
-                        </div>
-                    </div>
+        <div id="content" className="animated fadeInDown">
+            <h3 className="text-center text-danger">My Profile</h3><hr/>
+            <div className="col-lg-7 col-md-6 text-center" style={boxShadowStyle}>
+                <br/>
+                <img src="assets/img/avatars/sunny.png" style={bigCirclePhotoStyle} />
+                <h1 className="text-danger"><br/>{props.user.firstname + " "}<span className="semi-bold">{props.user.lastname}</span>
```
Comment (lmartin73) [inline src/app/routes/profile/components/profile.js]: perhaps we should just map the last name to and so forth to it's own prop. that way this presentational view doesn't have to know about the user structure in the store.

File: src/app/routes/profile/containers/editprofile.js [hunk 4]
```diff
@@ -219,147 +220,129 @@ class EditProfile extends React.Component {
...
+                <h3 className="text-center text-danger">Edit Profile</h3><hr/>
+                    <div className="col-lg-10 col-lg-offset-1 col-md-10 col-md-offset-1" style={boxShadowStyle}>
+                        <UiValidate options={this.validationOptions}>
+                            <form id="checkout-form" className="smart-form" noValidate="novalidate">
+                                <fieldset>
+                                    <div className="row">
+                                        <div className="col-lg-4 col-md-4">
+                                            <div className="text-center">
+                                                <div className="">
+                                                    <img className="img-thumbnail" name="image" ref="image" style={bigCirclePhotoStyle} src={this.state.photoURL}/><br/>
```
Comment (lmartin73) [inline src/app/routes/profile/containers/editprofile.js]: should this be 'this.profile.photoURL' as well as for the following places where you try to access the profile data from the local state?

**Review by lmartin73 — CHANGES_REQUESTED** [review]

File: src/app/components/styles/styles.js (hunk 1 above)
Comment (jhubbard2017) [reply src/app/components/styles/styles.js:1]: I would think so, since its gonna be access by many inputs

Comment (lmartin73) [reply src/app/components/styles/styles.js:1]: ok

**Review by lmartin73 — APPROVED** [review]

**Pull request merged**